docs/Documentation/05-linking-between-articles.md

```diff
@@ -4,14 +4,18 @@ tags: []
 
 # Linking Between Articles
 
-## What
+Create hyperlinks to other Markdown files to create additional navigation within your documentation. All links are relative to the current file.
 
-Create hyperlinks to other Markdown files to create additional navigation within your documentation.
+1. Select the **Markdown file** you wish to modify
 
-## How
+2. Right click the file you're linking to and select "Copy Relative Path." To reference an element from within the API specification, first build the relative path to the `api.yaml` or `api.json` file (ie `../reference/api.yaml`) and then the path to the API endpoint call. *Note: copying the path does not include the HTTP request method. You will need to append that yourself.*
```

Comment:
Also if you navigate to the openapi file, you can copy the path to the API endpoint. You'll still need to make it relative to your markdown article though, but you can combine it with the "Copy Relative Path" method you mentioned here.

Example output from "Copy Path" on an operation:
```
reference/todos/openapi.v1.json/paths/~1todos~1{todoId}
```

![image](https://user-images.githubusercontent.com/7423098/151426440-485ad320-a605-49d0-8a66-fb43710e2fd4.png)

Reply:
I think one thing that's super annoying is that it doesn't allow you to select the HTTP method. Selecting the path without the method often means you end up at a dead link. Can this get fixed so that it'll copy the path to the first method on the endpoint by default, but if you right click on the 'put' or 'delete' it'll copy those methods?

Reply:
When is "Copy Path" useful though? If relative is the only way it works, it doesn't add anything to have the absolute path accessible.

Reply:
> When is "Copy Path" useful though? If relative is the only way it works, it doesn't add anything to have the absolute path accessible.

Absolute paths are working in published docs, but it looks like they're broken in Studio. We'll get that fixed!

> I think one thing that's super annoying is that it doesn't allow you to select the HTTP method. Selecting the path without the method often means you end up at a dead link. Can this get fixed so that it'll copy the path to the first method on the endpoint by default, but if you right click on the 'put' or 'delete' it'll copy those methods?

Yeah I like this idea too. 

I also think "Copy Relative Path" for an Operation should just give you the path relative to the openapi file: `/paths/~1todos~1{todoId}/get`